.github/ISSUE_TEMPLATE/NEW_RELEASE.md

```diff
@@ -50,12 +51,17 @@ Please do not remove items from the checklist
 - [ ] For major or minor releases, merge the `main` branch into the `website` branch to publish the updated documentation.
 - [ ] Send an announcement email to `sig-scheduling@kubernetes.io` and `wg-batch@kubernetes.io` with the subject `[ANNOUNCE] kueue $VERSION is released`.   <!--Link: example https://groups.google.com/a/kubernetes.io/g/wg-batch/c/-gZOrSnwDV4 -->
 - [ ] Update the below files with respective values in `main` branch :
-  - Latest version in `README.md`
-  - Latest version in `cmd/experimental/kjobctl/docs/installation.md`
+  - Latest version in:
+    - `README.md`
+    - `Makefile`
```

Comment:
just the makefile, and then run `make prepare-release-branch`

Reply:
yeah, I did it all manually

Reply:
Updated the steps, PTAL